src/pmpo_c.cpp

```diff
@@ -699,6 +699,48 @@ void polympo_getMeshOnSurfDispIncr_f(MPMesh_ptr p_mpmesh, const int nComps, cons
   Kokkos::deep_copy(arrayHost, array_d);
 }
 
+void polympo_setMeshVtxStrainRate_f(MPMesh_ptr p_mpmesh, const int nVertices, const double *xNormal, const double *yNormal, const double *zNormal, const double *xyShear, const double *xzShear, const double *yzShear){
```

Comment:
Need to discuss if each component (of the 6 components of symmetric tensor) will be given as a separate array.

Reply:
It has been converted to take one 2d array instead